### ISSUE TITLE
Cannot control volume for Dell U3219Q

### ISSUE THREAD

**Issue by oalper** [MonitorControl/MonitorControl#1300]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

Hi,

Brightness is working fine, but Volume control from the menu (screenshot attached below) has no effect whatsoever, muting,increasing/decresaing from the menu does not help.

Re. my setup:

-> M1 Max Macbook Pro (14-inch, 2021)
-? Macos: 13.2
-> Monitor Control Version: 4.1.0
-> Accessibility access granted
-> Monitor : U2319Q, trying with a regular apple wired headphone

### Expected behavior

Again, VOLUME control has no effect, it does not do anything

### Anything else?

![image](https://user-images.githubusercontent.com/1282274/215310727-93d02056-e243-4660-a4f7-c6f7d50a7888.png)
![image](https://user-images.githubusercontent.com/1282274/215310740-65e12f88-7729-4fb1-8743-9f280b59c9cc.png)

### Environment Information (please complete the following information)

```markdown
- macOS version: 13.2
- Mac model: M1 Max Macbook Pro (14-inch, 2021)
- MonitorControl version: 4.1.0
- Monitor(s): Dell U3219Q
- Monitor Cable(s)/Connection(s): Macbook Pro -> USB-C -> Dell -> Regular / wired iPhone headphone
- Apple Silicon/M1 (yes or no): YES
```

**Comment by waydabber** [comment]
Some displays simply don't support DDC volume controls, several Dells, Samsungs, Lenovos has this issue. Some support it via one input but not with the other. Did you try the display with a different app as well just to make sure?

**Comment by oalper** [comment]
Tried one more app (I think it was better display), same thing happened :(

**Comment by waydabber** [comment]
Yes, probably the display does not support volume controls. No need to despair, you can install other apps that use a software technique (adjust the signal level in the audio stream with an audio driver filter). Try eqMac or similar apps!
